tasmota/xdrv_55_touch.ino

```diff
@@ -317,5 +317,9 @@ bool Xdrv55(uint8_t function) {
   }
   return result;
 }
-
+#else
+// dummy for LVGL without a touch controller
+uint32_t Touch_Status(uint32_t sel) {
+return 0;
+}
 #endif  // USE_TOUCH
```

Comment:
Where is the beginning of this code? 

There are so many defines in this file that it is impossible to find the corresponding ``#ifdef``

Reply:
Ah. Thanks. I now see it's a function. Stupid me.